tensorbay/cli/branch.py

```diff
@@ -11,12 +11,13 @@
 
 from ..client.gas import DatasetClientType
 from .tbrn import TBRN, TBRNType
-from .utility import error, get_dataset_client, get_gas, shorten
+from .utility import deploy_request_config, error, get_dataset_client, get_gas, shorten
 
 
 def _implement_branch(
     obj: Dict[str, str], tbrn: str, name: str, verbose: bool, is_delete: bool
 ) -> None:
+    deploy_request_config()
```

Comment:
Only use `deploy_request_config()` in `_implement_cli`.

Reply:
Yesir~